resourcepolicies.md

```diff
@@ -10,34 +10,298 @@ As we don't have yet an use case to iterate over all the existent resource polic
+* resource: optional, limit the returned policies to the ones related to the specified resource
+
+It returns the list of matching resource policies
+
+Return codes:
+* 200 OK - if the operation succeed
+* 401 Forbidden - if you are not authenticated
+* 403 Unauthorized - if you are not logged in with sufficient permissions. Only system administrators, users member of the group specified in the uuid parameter or user with ADMIN permission over it can use the endpoint
+* 422 UNPROCESSABLE ENTITY - if the uuid parameter is missing or invalid
+
+#### authorizeAnyOf
+**/api/authz/resourcepolicies/search/authorizeAnyOf?resource=<:resource>&action=<:action>&action=<:actionN>[&useInheritance=<:boolean>]**
+
+The supported parameters are:
+* page, size [see pagination](README.md#Pagination). MUST be size = 0, the endpoint *currently* never return a list of resourcepolicies, in future it could be possible to return the list of resourcepolicy that collectively grant the current user the requested actions
+* resource: mandatory, the uuid of the resource that need to be checked for permission
+* action: multiple, at least one required. The actions that need to be verified
+* useInheritance: optional, assume false as default. True mean that policies on parents objects are considered, false will focus only on policy attached directly to the resource
+
+It would returns the list of resource policies that grant to the user the requested actions of the resource. When size = 0 the pagination totalElements attribute in the response can be used to discriminate if all the actions are allowed or not.
+If allowed the totalElements attribute will be 1 otherwise 0.  
+
+Return codes:
+* 200 OK - if the operation succeed. The totalElements attribute in the pagination information will be 1 if the actions are granted to the current user, otherwise 0.
+* 401 Forbidden - if you are not authenticated
+* 422 UNPROCESSABLE ENTITY - if the resource and/or action parameters are missing or invalid (including the case of unexisting resource uuid)
+
+## Creating a resource policy
+**POST /api/authz/resourcepolicies?resource=<:uuid>&[eperson=<:eperson>|group=<:group>]**
+
+Request body:
+```json
+{
+  "name": null,
+  "description": null,
+  "policyType": "TYPE_SUBMISSION",
+  "action": "READ",
+  "startDate" : null,
+  "endDate" : null,
+  "type": "resourcepolicy"
+}
+```
+
+The parameters are:
+* resource: mandatory, the uuid of the resource target of the policy
+* eperson: the eperson that will be grant of the permission. Exactly one of eperson or group is required
+* group: the group that will be grant of the permission. Exactly one of eperson or group is required
+
+The json body must be valid that mean
+- type must be resourcepolicy
+- policyType must be one of the value specified in the GET description
+- action must be one of the value specified in the GET description
+- startDate / endDate if present must respect the format specified in the GET description
```

Comment:
Can you add return codes, error handling, permissions required?

Reply:
done